samples/onnx_accelerator_sample1/onnxacceleratorsampleone/with_ggv2/build_deployment_package.py

```diff
@@ -0,0 +1,117 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of
+# this software and associated documentation files (the "Software"), to deal in
+# the Software without restriction, including without limitation the rights to
+# use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+# the Software, and to permit persons to whom the Software is furnished to do so.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+# FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+# COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+# IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import torch
+import tarfile
+import boto3
+import os
+import json
+import yaml
+
+# first we need to retrieve the model pth file, for that let's consult the model package
+model_package_arn = os.environ["MODEL_PACKAGE_ARN"]
+deployment_bucket_name = os.environ['DEPLOYMENT_BUCKET_NAME']
+region = os.environ["AWS_REGION"]
+
+client_sm = boto3.client("sagemaker")
+
+def update_component_config_in_json_file(filepath, version, component_name):
+    with open(filepath+'gdk-config.json') as f:
+        data = json.load(f)
+
+        data['component'][component_name]['publish']['bucket'] = data['component'][component_name]['publish']['bucket'].replace('_BUCKET_NAME_', deployment_bucket_name)
+        data['component'][component_name]['publish']['region'] = data['component'][component_name]['publish']['region'].replace('_REGION_', region)
+        data['component'][component_name]['version'] = data['component'][component_name]['version'].replace('_COMPONENT_VERSION_', version)
+
+        with open(filepath+'gdk-config.json', 'w') as f:
+            json.dump(data, f)
+
+def update_component_recipe_yaml_file(filepath, version, component_name, model_name):
+    with open(filepath+"recipe.yaml", 'r') as stream:
+        try:
+            loaded = yaml.safe_load(stream)
+        except yaml.YAMLError as exc:
+            print(exc)
+
+    # Modify the fields from the dict
+    if "detector" in filepath:
```

Comment:
This should not be needed, since the model to use is determined by which model component you are deploying